tests/consensus_tests/test_shard_wal_delta_transfer.py

```diff
@@ -39,12 +40,17 @@ def check_data_consistency(data):
         data_j = data[j]
 
         if data_i != data_j:
-            ids_i = set(x.id for x in data_i["points"])
-            ids_j = set(x.id for x in data_j["points"])
+            ids_i = set(x['id'] for x in data_i["points"])
+            ids_j = set(x['id'] for x in data_j["points"])
```

Comment:
It is not entirely clear to me why `.id` worked in most cases (?), and suddenly started to fail in your branch.

Reply:
No idea either :shrug: 

I suspect the code has not been exercised in a very long time.
This object is now clearly a dictionary.